src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Registry.UnitTest/Helper/RegistryHelperTest.cs

```diff
@@ -31,7 +31,7 @@ public void GetRegistryBaseKeyTestOnlyOneBaseKey(string query, string expectedBa
         [TestMethod]
         public void GetRegistryBaseKeyTestMoreThanOneBaseKey()
         {
-            var (baseKeyList, _) = RegistryHelper.GetRegistryBaseKey("HKC\\Control Panel\\Accessibility"); /* #no-spell-check-line */
+            var (baseKeyList, _) = RegistryHelper.GetRegistryBaseKey("HKC\\Control Panel\\Accessibility");
```

Comment:
I'm not sure why this line got this comment. I didn't look into its history, but it didn't seem worth it.